package/MDAnalysis/core/groups.py

```diff
@@ -995,6 +995,11 @@ def select_atoms(self, sel, *othersel, **selgroups):
                 range consists of two numbers separated by a colon (inclusive)
                 such as ``resid 1:5``. A residue number ("resid") is taken
                 directly from the topology.
+                If icodes are present in the topology, then these will be
+                taken into account.  Ie 'resid 163B' will only select resid
+                163 with icode B while 'resid 163' will select only residue 163.
```

Comment:
I thought we always select the first residue with number 163 now. With what ever icode that happens to be. So the docs could say that 'resid 163' will select the first residue in the topology. While one can also specify icodes to select specific residues with the same number.

But what about PDB files like this

```
ATOM     34  CA AARG A   2C      12.353  85.696  94.456  0.50 36.67           C
ATOM     35  CA BARG A   2A      12.333  85.862  95.041  0.50 36.42           C
```

Where the first icode lines aren't sorted alphabetically? Which atom would `resid 2` select in this case?